src/MMIXWare.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="MMIXware"
+   name="MMIXware License" listVersionAdded="3.22">
+      <crossRefs>
+         <crossRef>https://gitlab.lrz.de/mmix/mmixware/-/blob/master/boilerplate.w</crossRef>
+      </crossRefs>
+      <text>
+         <list>
+            <item>
+               <p>
+                  copyright 1999 Donald E. Knuth This file may be freely copied and
```

Comment:
copyright notice needs to be enclosed in copyrightText tag

Reply:
I checked how other licenses in src/ do that and I do that similarly. Tell me if needs to be done differently.